elfpy/data/db_schema.py

```diff
@@ -24,21 +25,21 @@ class PoolConfig(Base):
 
     __tablename__ = "poolconfig"
 
-    contractAddress: Mapped[str | None] = mapped_column(String, primary_key=True)
-    baseToken: Mapped[str | None] = mapped_column(String, default=None)
-    initializeSharePrice: Mapped[float | None] = mapped_column(Numeric, default=None)
-    positionDuration: Mapped[int | None] = mapped_column(Integer, default=None)
-    checkpointDuration: Mapped[int | None] = mapped_column(Integer, default=None)
-    timeStretch: Mapped[float | None] = mapped_column(Numeric, default=None)
-    governance: Mapped[str | None] = mapped_column(String, default=None)
-    feeCollector: Mapped[str | None] = mapped_column(String, default=None)
-    curveFee: Mapped[float | None] = mapped_column(Numeric, default=None)
-    flatFee: Mapped[float | None] = mapped_column(Numeric, default=None)
-    governanceFee: Mapped[float | None] = mapped_column(Numeric, default=None)
-    oracleSize: Mapped[int | None] = mapped_column(Integer, default=None)
-    updateGap: Mapped[int | None] = mapped_column(Integer, default=None)
-    invTimeStretch: Mapped[float | None] = mapped_column(Numeric, default=None)
-    termLength: Mapped[float | None] = mapped_column(Numeric, default=None)
+    contractAddress: Mapped[Union[str, None]] = mapped_column(String, primary_key=True)
```

Comment:
let's add a comment at the top of this file that we are using Union to support 3.9 and link that GH issue.

Reply:
good call -- thanks @sentilesdal